### PR TITLE
Change strotime input to correctly render 'last week'

### PR DESCRIPTION
**Pull request by lgrn**
This works fine on mondays and tuesdays, but breaks for all other days
of the week, likely due to the 'midnight -1 sec' rescuing monday and
tuesday by actually making them sunday and monday, but the 'monday'
passed for time_end then becomes monday next week, causing the report to
show everything up until now.

This commit changes the overly verbose strtotime inputs to something
that hopefully more accurately should reflect "last week" regardless of
what day of the week it is.

For testing/reproduction, I suggest disabling NTP and manually changing
the OS time, then refreshing a report to see how it changes with the
days.

This fixes MON-12548 which has more information in Jira.

### CODE REVIEW

**Comment by lgrn** [comment]
I've tested this by grabbing `monitor-ninja-2020.k.1-p9.g164e389.20210119125354.el7.x86_64.rpm` from the build and installing it on a clean 8.2.5 install, it seems to fix the issue. I've also added new tests that are more expansive and should better catch future errors.

File: test/report_Test.php [hunk 1]
```diff
@@ -1082,13 +1082,25 @@ public function time_input_for_report_options() {
...
+				'2015-08-02 23:59:59'
+			),
+			array(
+				'lastweek',
+				'2015-08-09 17:00:00', // a Sunday
+				'2015-07-27 00:00:00',
+				'2015-08-02 23:59:59'
 			),
 			array(
 				'last7days',
```
Comment (erikdsjostrom) [inline test/report_Test.php:1106]: Why not include Tuesday, Thursday and Saturday as well?

**Review by erikdsjostrom — COMMENTED** [review]
Looks good to me, had one minor comment.

**Review by erikdsjostrom — APPROVED** [review]
Looks good to me! Just remember to squash the commits! :+1:

**Pull request merged**